[
    {
        "repo_name": "libssh2/libssh2",
        "CVE_ID": "CVE-2020-22218",
        "Problem_Type": "CWE-787",
        "Description": "An issue was discovered in function _libssh2_packet_add in libssh2 1.10.0 allows attackers to access out of bounds memory.",
        "URL": "https://github.com/libssh2/libssh2/pull/476",
        "Tag": "['Issue Tracking', 'Patch']",
        "fix_commit_hash": "642eec48ff3adfdb7a9e562b6d7fc865d1733f45",
        "inducing_commit_hash_pyszz": [],
        "inducing_commit_hash_pd": [
            "9d55db6501aa4e21f0858cf36cdc2ddc11b96e83"
        ],
        "matched": []
    }
]